docs/migrate-to-nativecurlclient.md

```diff
@@ -0,0 +1,255 @@
+# Migrate from `Redmine\Client` to `Redmine\Client\NativeCurlClient`
+
+Since `php-redmine-api` v1.8.0 there is a new native cURL client
+`Redmine\Client\NativeCurlClient`. This guide will help you to migrate your code
+to this client.
```

Comment:
```suggestion
Since `v1.8.0`, there is a new native cURL client `Redmine\Client\NativeCurlClient`. This guide will help migrate your code to this client.
```